src/pages/documentation/faq.md

```diff
@@ -84,20 +84,37 @@ iOS
+| :------------------------ | :------- | :----------- | :------------------------------------------------ |
+| Core                      | 925      | 497          | Android only: `kotlin-stdlib`                     |
+| Identity                  | 136      | 51           |                                                   |
+| Lifecycle                 | 150      | 59           |                                                   |
+| Signal                    | 18       | 33           |                                                   |
+| Edge Network extension    | 253      | 99           | Identity for Edge Network                         |
+| Consent for Edge Network  | 33       | 25           | Edge Network extension, Identity for Edge Network |
+| Identity for Edge Network | 69       | 49           |                                                   |
+| Analytics                 | 133      | 95           |                                                   |
+| Assurance                 | 917      | 527          | Android only: `androidx.constraintlayout`         |
+| Audience                  | 82       | 44           | Identity                                          |
+| Campaign                  | 166      | 80           | Identity                                          |
+| Campaign Classic          | 34       | 42           | Identity                                          |
+| Edge Bridge               | 0.84     | 6            |                                                   |
+| Media                     | 283      | 137          |                                                   |
+| Messaging                 | 33       | 77           | Edge Network extension, Identity for Edge Network |
+| Optimize                  | 88       | 80           | Edge Network extension, Identity for Edge Network |
+| Places                    | 799      | 59           |                                                   |
+| Target                    | 283      | 99           |                                                   |
+| User Profile              | 18       | 29           |                                                   |
+
+The sizes in the table are provided for estimation purposes only, with the following considerations:
+
+* †Mobile Core, which includes the Rules Engine and Services extensions, is required for all other extensions. The final app size increase can be calculated by adding the Mobile Core size to each of the enabled extensions. 
+  * For example, the iOS app distribution using the Target and Analytics extensions will have a total size increase of 1341 KB. (Core: 925 KB + Analytics: 133 KB + Target: 283 KB).
+* The iOS (SDK extension versions 3+) estimates are based on [Google's CocoaPods Size Measurement tool](https://github.com/google/cocoapods-size). 
+* The Android (SDK extension versions 2+) size estimates listed refer to unsigned apps and do not account for applying ProGuard.
```

Comment:
since this answer will sit in the latest version docs, can we remove SDK versions 3+, respectively 2+?